app/static/lib/conp-react/umd/1.conp-react.js

```diff
@@ -4,7 +4,7 @@
  */
 (window["webpackJsonpCONPReact"] = window["webpackJsonpCONPReact"] || []).push([[1],{
 
-/***/ 26:
+/***/ 29:
```

Comment:
Will these local changes have any substantial effect?  I guess this is because we are including the compiled react library in the repo.  Maybe we should consider making compiling the react components a part of the installation procedure.